src/pages/workspaces/workspace/analysis/modals/ComputeModal/AzurePersistentDiskSection.ts

```diff
@@ -0,0 +1,36 @@
+import { div } from 'react-hyperscript-helpers';
```

Comment:
Miguel's input - is this file useful? Should we combine with it's children?

Reply:
Breaking out components is always nice... but we do have quite a few files in modals/ComputeModal folder now.... I would consider breaking these up and shove the simpler sub-components into azure-compute-modal-components and gcp-compute-modal-components respectively.... or just split into AzureComputeModal and GcpComputeModal sub-folders, and only keep shared modules between them at the modals/ComputeModal level....

Yeah, I think I like the last option best.

Reply:
Totally agree. Jacob was saying the same thing.

I should still keep the file naming prepended with it's type, correct?

Aka the AzurePersistentDiskSizeSelectInput should still contain the Azure